pages/sections/landing.tsx

```diff
@@ -10,7 +10,215 @@ import SignupCounter from '@/components/SignupCounter';
 import Link from 'next/link';
 import { useRouter } from 'next/router';
 import Image from 'next/image';
-import Skyline from '../../public/assets/landing.svg';
+import Skyline from '../../public/assets/skyline.svg';
+import cloud from '../../public/assets/cloud.svg';
+import airplane from '../../public/assets/airplane.svg';
+import bird from '../../public/assets/bird.svg';
+
+export const Assets = () => {
+  return (
+    <>
+      {/* cloud  left low opacity*/}
```

Comment:
you know how th overlapping clouds are moving separately. Marketing wants those 2 clouds to be grouped together. hope that makes sense, but lmk if u need further clarifications